File: tirefitting/modules/core/src/com/company/tirefitting/core/TireDismount.java
package com.company.tirefitting.core;

import org.springframework.stereotype.Component;

@Component(TireDismount.NAME)
public class TireDismount implements RequestTypeDurationCalculator {
    public static final String NAME = "tirefitting_TireDismount";

    @Override
    public double getDuration(int radius) {
        return 60 * 60 * (1 + ((double)radius - 13) / 5);
    }
}

File: tire_fitting/lib/requestTypes/TireDismount.dart
import 'package:tire_fitting/requestTypes/RequestType.dart';

class TireDismount extends RequestType{
  @override
  double getDuration(int radius) {
    return 60 * 60 * (1 + (radius - 13) / 5);
  }

  @override
  String name = "TIRE_DISMOUNT";

}